tests/test_middleware.py

```diff
@@ -198,7 +198,7 @@ def __call__(self, to_provide):
         # we're returning a class that's not listed in self.provided_classes
         return {
             ExtraClassData: ExtraClassData("this should be returned"),
-            ResponseData: ResponseData("example.com", "this shouldn't"),
+            HttpResponse: HttpResponse("example.com", "this shouldn't"),
```

Comment:
body type should be bytes, it raises an exception if str is passed: https://github.com/scrapinghub/web-poet/blob/256a0c3af97d9d91a3e2150b461d2d723b375f32/tests/test_page_inputs.py#L73

Reply:
Good catch! Updated this on https://github.com/scrapinghub/scrapy-poet/pull/67/commits/64263a9ab64e0d672eae10132f506ff911f6eacd